packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -65,6 +65,43 @@ import 'theme.dart';
 /// ```
 ///
 ///{@end-tool}
+///
+/// {@tool snippet}
+/// This example mirrors the previous one, creating the same list using the
+/// [ReorderableListView.builder] constructor. Using the [IndexedWidgetBuilder], The
+/// list items are built lazily on demand.
+/// ```dart
+/// final List<int> _items = List<int>.generate(50, (int index) => index);
+///
+/// @override
+/// Widget build(BuildContext context){
+///   final ColorScheme colorScheme = Theme.of(context).colorScheme;
+///   final Color oddItemColor = colorScheme.primary.withOpacity(0.05);
+///   final Color evenItemColor = colorScheme.primary.withOpacity(0.15);
+///
+/// return ReorderableListView.builder(
+///   padding: const EdgeInsets.symmetric(horizontal: 40),
+///   itemCount:_items.length,
+///   itemBuilder: (BuildContext _, int index) {
```

Comment:
Would prefer `context` instead of `_` here.

Reply:
changed ```_``` to ``` context```